### PR TITLE
imgtools.mstools notebook with mscombine example

### PR DESCRIPTION
**Pull request by bourque**
This notebook contains a Python alternative to IRAF's `imgtools.mstools.mscombine` task using WFC3/UVIS data.

@SaraOgaz If you go through with this, I can send some test data your way.

### CODE REVIEW

**Comment by justincely** [comment]
thanks @bourque.  @SaraOgaz is gonna take a look over and may have some changes for you to make.

**Comment by justincely** [comment]
@bourque, i'm going to review this while @SaraOgaz is away.  Could i ask you to rebase with the current master to get current first?

**Comment by SaraOgaz** [comment]
@bourque, sorry I never got a chance to look at this Matt  :(

**Comment by bourque** [comment]
@justincely I hit the rebase button, I think you have to approve it.

**Comment by justincely** [comment]
thanks @bourque.  Looks like it rabased fine - i'll take a look and review today.

**Comment by SaraOgaz** [comment]
This looks good to me, but I was just chatting with @bourque about one thing.  For image combination we've used ``ccdproc`` for some of the other tasks.  But I see no reason we couldn't have two examples for this task, use Matt's from scratch example, and and add another using ``ccdproc``.  What do you think @justincely? 

The one other thing is we should wait until after 0.1 release to add this notebook.  So @bourque, you can probably wait to rebase again until we get there.

**Comment by justincely** [comment]
@SaraOgaz @bourque sure thing - i see no reason why we can't have two examples for now.  If one becomes the dominant, or considerably easier/better, then we can get rid of one later on down the road.  

I'm fine to put this off until after initial release.  Sara - can you review this with a look towards formatting of the notebook? Trailer, construction workers, etc so we get a consistent look?

File: docs/source/notebooks/imgtools.mstools.ipynb [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    "* [mscombine](#mscombine)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='mscombine'></a>\n",
+    "\n",
+    "## mscombine"
```
Comment (SaraOgaz) [inline docs/source/notebooks/imgtools.mstools.ipynb]: Only change in here is that we added the "Please review the Notes section above...." below each task header.

**Comment by bourque** [comment]
@SaraOgaz Cool, thanks. I've added the comment about the Notes section and rebased.

**Pull request merged**